### ISSUE TITLE
Error With MySQL database when merging claims

### ISSUE THREAD

**Issue by FredoJF** [Xyness/SimpleClaimSystem#6]
Got this error when trying to merge two claims. Plugin is version 1.10, database created in 1.9. Using latest Paper 1.21

```
[18:09:37 WARN]: java.sql.SQLException: Data truncated for column 'X' at row 8
[18:09:37 WARN]:        at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:113)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:590)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:335)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1106)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1045)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1369)
[18:09:37 WARN]:        at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1030)
[18:09:37 WARN]:        at SimpleClaimSystem.jar//com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
[18:09:37 WARN]:        at SimpleClaimSystem.jar//com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
[18:09:37 WARN]:        at SimpleClaimSystem.jar//fr.xyness.SCS.ClaimMain.lambda$mergeClaims$250(ClaimMain.java:3746)
[18:09:37 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
[18:09:37 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
[18:09:37 WARN]:        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
[18:09:37 WARN]:        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
[18:09:37 WARN]:        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
[18:09:37 WARN]:        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
[18:09:37 WARN]:        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
```

**Comment by Xyness** [comment]
Hey, thank's for reporting it, i'll check!

**Comment by Xyness** [comment]
Hey @FredoJF its fixed ! The update (v1.10.0.1) is available :) Thank's!
